<launch>

    <rosparam command="load" file="$(find phoenix_bridge)/config/interface_description.yaml" />

    <node pkg="phoenix_bridge" name="phoenix_io_services_node" type ="phoenix_io_services" output="screen">
    </node>

    <rosparam command="load" file="$(find phoenix_bridge)/config/liveliness_config.yaml" />

    <node pkg="phoenix_bridge" name="liveliness_check" type ="liveliness_check" output="screen">
    </node>

</launch>
